### PR TITLE
Update logrus to 0.6.6

### LINKED ISSUE #11252: Problem caused by old vendor/src/xxx version
Hi, how often do docker update its vendor/src/xxx?
Recently, I am doing some further developing on logrus, and things go wrong because of the old version of vendor/src/github.com/Sirupsen/logrus. So can docker update logrus and other third-party applications? Thanks!
@tianon , @unclejack , @tiborvass , @jpetazzo

### PR DESCRIPTION
**Pull request by jessfraz**
closes #11252

### CODE REVIEW

**Comment by jfrazelle** [comment]
what did I miss? is it because libcontainer has 0.6.0

```
$ ./project/vendor.sh 
github.com/kr/pty @ 05017fcccf: rm old, clone, rm VCS, done
github.com/gorilla/context @ 14f550f51a: rm old, clone, rm VCS, done
github.com/gorilla/mux @ 136d54f81f: rm old, clone, rm VCS, done
github.com/tchap/go-patricia @ v1.0.1: rm old, clone, rm VCS, done
code.google.com/p/go.net @ 84a4013f96e0: rm old, clone, rm VCS, done
code.google.com/p/gosqlite @ 74691fb6f837: rm old, clone, rm VCS, done
github.com/docker/libtrust @ 230dfd18c232: rm old, clone, rm VCS, done
github.com/Sirupsen/logrus @ v0.6.6: rm old, clone, rm VCS, done
github.com/go-fsnotify/fsnotify @ v1.0.4: rm old, clone, rm VCS, done
github.com/docker/libcontainer @ dd3cb8822352fd4acc0b8b426bd86e47e98f6853: rm old, clone, rm VCS, done
github.com/coreos/go-systemd @ v2: rm old, clone, rm VCS, done
github.com/godbus/dbus @ v2: rm old, clone, rm VCS, done
github.com/Sirupsen/logrus @ v0.6.0: rm old, clone, rm VCS, done
github.com/syndtr/gocapability @ e55e583369: rm old, clone, rm VCS, done
```

**Comment by LK4D4** [comment]
@jfrazelle Yup, you can remove it like `codegangsta/cli`

**Comment by jfrazelle** [comment]
fixed cool

**Comment by jfrazelle** [comment]
cool it succeeded ping @tianon

File: vendor/src/github.com/Sirupsen/logrus/README.md [hunk 1]
```diff
@@ -307,7 +324,7 @@ The built-in logging formatters are:
 
 Third party logging formatters:
 
-* [`zalgo`](https://github.com/aybabtme/logzalgo): invoking the P͉̫o̳̼̊w̖͈̰͎e̬͔̭͂r͚̼̹̲ ̫͓͉̳͈ō̠͕͖̚f̝͍̠ ͕̲̞͖͑Z̖̫̤̫ͪa͉̬͈̗l͖͎g̳̥o̰̥̅!̣͔̲̻͊̄ ̙̘̦̹̦.
+* [`zalgo`](https://github.com/aybabtme/logzalgo): invoking the P͉̫o̳̼̊w̖͈̰͎e̬͔̭͂r͚̼̹̲ ̫͓͉̳͈ō̠͕͖̚f̝͍̠ ͕̲̞͖͑Z̖̫̤̫ͪa͉̬͈̗l͖͎g̳̥o̰̥̅!̣͔̲̻͊̄ ̙̘̦̹̦.
```
Comment (tiborvass) [inline vendor/src/github.com/Sirupsen/logrus/README.md]: i think this is what fixed it all :P

**Comment by tiborvass** [comment]
LGTM

File: vendor/src/github.com/Sirupsen/logrus/README.md (hunk 1 above)
Comment (jfrazelle) [inline vendor/src/github.com/Sirupsen/logrus/README.md]: lol

**Comment by cpuguy83** [comment]
LGTM

**Comment by tianon** [comment]
Does libcontainer not need any fixes for the newer logrus?

**Comment by jfrazelle** [comment]
i can make a pr there
but no i dont believe it does

**Pull request merged**